### PR TITLE
little typos

### PR DESCRIPTION
**Pull request by mildsunrise**

### CODE REVIEW

File: tests/incomingstreamtrack.js [hunk 1]
```diff
@@ -51,8 +51,8 @@ tap.test("IncomingMediaStream::create",async function(suite){
 		const ssrcs = videoTrack.getSSRCs();
 		//Check them
 		test.equals(Object.keys(ssrcs).length,1);
-		test.equals(ssrcs[""].media.ssrc,media);
-		test.equals(ssrcs[""].rtx.ssrc,rtx);
+		test.equals(ssrcs[""].media,media);
+		test.equals(ssrcs[""].rtx,rtx);
```
Comment (mildsunrise) [inline tests/incomingstreamtrack.js:55]: @murillo128 can you clarify if `getSSRCs()` is supposed to return the SSRCs or the RTPIncomingSources? I'm quite confused

File: lib/IncomingStreamTrack.js [hunk 2]
```diff
@@ -359,11 +362,11 @@ class IncomingStreamTrack extends Emitter
 			if (id)
 			{
 				//Create simulcast info
-				const encodingInfo = new TrackEncodingInfo(id);
+				const encodingInfo = new TrackEncodingInfo(id, false);
```
Comment (murillo128) [inline lib/IncomingStreamTrack.js:365]: can we add a default to the `TrackEncodingInfo`  so `paused=false` and not have to specify it when calling the constructor?

File: tests/incomingstreamtrack.js (hunk 1 above)
Comment (murillo128) [reply tests/incomingstreamtrack.js:55]: should be an ssrc array, not the incoming sources.

File: lib/IncomingStreamTrack.js (hunk 2 above)
Comment (mildsunrise) [reply lib/IncomingStreamTrack.js:365]: done

**Pull request merged**